hotxlfp/grammarparser/parser.py

```diff
@@ -79,9 +79,18 @@ def p_expression_arithmetic_operator(self, p):
                   | expression_paren MINUS expression_paren
                   | expression MINUS expression_paren
                   | expression_paren MINUS expression
+                  | expression_paren MULT expression_paren
                   | expression MULT expression
+                  | expression_paren MULT expression
+                  | expression MULT expression_paren
+                  | expression_paren DIV expression_paren
+                  | expression_paren DIV expression
+                  | expression DIV expression_paren
                   | expression DIV expression
                   | expression AMP expression
+                  | expression_paren CARET expression_paren
+                  | expression_paren CARET expression
+                  | expression CARET expression_paren
                   | expression CARET expression
```

Comment:
This is rather verbose now. Not sure if you have a better way here

Reply:
Nothing at the moment.